src/base/mesh_object.cu

```diff
@@ -0,0 +1,6 @@
+#include "mesh_object.h"
+
+gm::MeshObject::MeshObject(const std::shared_ptr<Mesh> &mesh,
```

Comment:
If you only have the constructor implementation in the .cu file I think we can remove this file and move it into the .h file. If more functions need to be added later, we can expand this out.

Reply:
Fixed in 5499c57